cloudinit/distros/opensuse.py

```diff
@@ -66,15 +74,72 @@ def install_packages(self, pkglist):
         )
 
     def package_command(self, command, args=None, pkgs=None):
+        pckg_op = command
```

Comment:
I'm not quite understanding the difference between `pckg_op` and `command`. Here we set `pckg_op` to `command`, but then later down we set `command` based on `pckg_op`. Is there a way to simplify that?

Reply:
This was need, I thought because, because I was basically checking for "command=='upgrade'" twice but in the second check it may have already been set to "update". This was a logic bug.  No longer needed.